src/main/java/sample/Controller.java

```diff
@@ -25,20 +26,21 @@
     private PasswordField password;
 
 
+    private static Session session = HibernateUtil.getSession();
     public void loginButtonIsClicked(ActionEvent event) throws IOException {
         User user = new User();
         user.setUsername(username.getText());
         user.setPassword(password.getText());
 
         UserVerifier userVerifier = new UserVerifier();
-        if (userVerifier.userExist(user)) {
+        if (userVerifier.userExist(user) && userVerifier.isUserEmployeeType(user)) {
             Parent parent = FXMLLoader.load(getClass().getResource("/employee.fxml"));
             Scene scene = new Scene(parent);
             Stage appStage = (Stage) ((Node) event.getSource()).getScene().getWindow();
             appStage.setScene(scene);
             appStage.show();
 
-        } else if (userVerifier.userExist(user) || userVerifier.isUserEmployeeType(user)) {
+        } else if (userVerifier.userExist(user) && !userVerifier.isUserEmployeeType(user)) {
```

Comment:
try to use the database interogation only once.  When you are calling userVerifier.userExists you also use the database